app/routes/info.py

```
@@ -69,11 +69,105 @@ async def _acquire_verify_lane_or_429() -> WorkloadToken:
+                # Don't cache size-cap skips or transient errors (same policy
+                # as the /dat/match-batch job).
+                if not result.get("reason") and not result.get("error"):
+                    await dat_store.set_match(path, result)
+                else:
+                    # Non-cacheable recompute (size cap / hash unavailable):
+                    # drop any stale prior row so /dat/matches/lookup doesn't
+                    # keep showing an outdated match after a (forced) rescan.
+                    await dat_store.delete_match(path)
+                if result.get("matched"):
+                    matched += 1
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)